Woodstar/Woodstar.Benchmark/Benchmarks.cs

```diff
@@ -0,0 +1,240 @@
+﻿using System.Data;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Diagnosers;
+using Microsoft.Data.SqlClient;
+using Woodstar.Tds;
+using Woodstar.Tds.Tds33;
+using Woodstar.Tds.Tokens;
+
+namespace Woodstar.Benchmark;
+
+[Config(typeof(Config))]
+public class Benchmarks
+{
+    const int Connections = 10;
+    class Config : ManualConfig
+    {
+        public Config()
+        {
+            Add(new SimpleJobAttribute(targetCount: 20).Config);
+            AddDiagnoser(MemoryDiagnoser.Default);
+            AddDiagnoser(ThreadingDiagnoser.Default);
+            AddColumn(new TagColumn("Connections", name => (name.EndsWith("Pipelined") ? 1 : Connections).ToString()));
+        }
+    }
+
+    const string EndPoint = "127.0.0.1:1433";
+    const string Username = "sa";
+    const string Password = "Abcd5678";
```

Comment:
This will avoid potential flagging

```suggestion
    const string Password = "PLACEHOLDER";
```

Reply:
Oops, already merged. Let's wait for the flagging :)